common/src/main/java/com/wynntils/screens/WynntilsDialogueHistoryScreen.java

```diff
@@ -0,0 +1,302 @@
+                    line,
+                    new TextRenderSetting(
+                            maxWidth,
+                            CommonColors.BLACK,
+                            HorizontalAlignment.Left,
+                            VerticalAlignment.Top,
+                            FontRenderer.TextShadow.NORMAL)));
+        }
+
+        FontRenderer.getInstance()
+                .renderTextsWithAlignment(
+                        poseStack,
+                        Texture.QUEST_BOOK_BACKGROUND.width() / 2f + 5,
+                        30,
+                        textRenderTaskList,
+                        maxWidth,
+                        Texture.QUEST_BOOK_BACKGROUND.height() - 50,
+                        HorizontalAlignment.Left,
+                        VerticalAlignment.Middle);
+    }
+
+    @Override
+    public boolean mouseScrolled(double mouseX, double mouseY, double delta) {
+        if (delta < 0) {
+            setCurrentPage(getCurrentPage() + 1);
+        } else if (delta > 0) {
+            setCurrentPage(getCurrentPage() - 1);
+        }
+
+        return true;
+    }
+
+    @SubscribeEvent
+    public void onQuestsReloaded(QuestBookReloadedEvent.DialogueHistoryReloaded event) {
+        this.setDialogues(QuestBookManager.getDialogueHistory());
+    }
+
+    private void renderTooltip(PoseStack poseStack, int mouseX, int mouseY) {
+        if (this.hovered instanceof ReloadButton) {
+            RenderUtils.drawTooltipAt(
+                    poseStack,
+                    mouseX,
+                    mouseY,
+                    100,
+                    RELOAD_TOOLTIP,
+                    FontRenderer.getInstance().getFont(),
+                    true);
+            return;
+        }
+
+        if (this.hovered instanceof QuestsPageButton) {
+            List<Component> tooltipLines = List.of(
+                    new TextComponent("[>] ")
+                            .withStyle(ChatFormatting.GOLD)
+                            .append(new TranslatableComponent(
+                                            "screens.wynntils.wynntilsDialogueHistory.questsPageButton.name")
+                                    .withStyle(ChatFormatting.BOLD)
+                                    .withStyle(ChatFormatting.GOLD)),
+                    new TranslatableComponent("screens.wynntils.wynntilsDialogueHistory.questsPageButton.description")
+                            .withStyle(ChatFormatting.GRAY),
+                    new TextComponent(""),
+                    new TranslatableComponent("screens.wynntils.wynntilsMenu.leftClickToSelect")
+                            .withStyle(ChatFormatting.GREEN));
+
+            RenderUtils.drawTooltipAt(
+                    poseStack,
+                    mouseX,
+                    mouseY,
+                    100,
+                    tooltipLines,
+                    FontRenderer.getInstance().getFont(),
+                    true);
```

Comment:
Perhaps extract this from the conditionals and instead set tooltipLines in the if statements

Reply:
I did not do this because there might be some special tooltip rendering where it is not always the same call. I will clean this up next commit / when I add mini-quest parsing.